<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>647</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>352</width>
    <height>647</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>352</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>QT微博客户端</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../Resources.qrc" >
    <normaloff>:/logo/images/qlisa.png</normaloff>:/logo/images/qlisa.png</iconset>
  </property>
  <property name="styleSheet" >
   <string notr="true" >QMainWindow > QWidget{&#xd;
	background-image: url(:/stylesheet/images/background1.png);&#xd;
	background-repeat: repeat-xy;&#xd;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QToolButton" name="toolButton_Logo" >
    <property name="geometry" >
     <rect>
      <x>5</x>
      <y>5</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor" >
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text" >
     <string>Logo</string>
    </property>
    <property name="iconSize" >
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable" >
     <bool>false</bool>
    </property>
    <property name="autoRepeat" >
     <bool>true</bool>
    </property>
    <property name="autoRaise" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_UserName" >
    <property name="geometry" >
     <rect>
      <x>65</x>
      <y>5</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor" >
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text" >
     <string>UserName</string>
    </property>
    <property name="toolButtonStyle" >
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
    <property name="autoRaise" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWebView" name="webView_Main" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>110</y>
      <width>352</width>
      <height>531</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement" >
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="url" >
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>40</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_userinfo" >
     <property name="spacing" >
      <number>1</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButton_Friends" >
       <property name="cursor" >
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="text" >
        <string>Friends</string>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Follower" >
       <property name="cursor" >
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="text" >
        <string>Followers</string>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_WeiboCount" >
       <property name="cursor" >
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="text" >
        <string>Statuses</string>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame" >
    <property name="geometry" >
     <rect>
      <x>1</x>
      <y>65</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_toolsbutton" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>4</number>
     </property>
     <property name="bottomMargin" >
      <number>4</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButton_NewWeiBo" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>写新微博</string>
       </property>
       <property name="text" >
        <string>New Status</string>
       </property>
       <property name="icon" >
        <iconset resource="../Resources.qrc" >
         <normaloff>:/toolButton/images/addnewweibo.png</normaloff>:/toolButton/images/addnewweibo.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_HomePage" >
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>我的首页</string>
       </property>
       <property name="text" >
        <string>HomePage</string>
       </property>
       <property name="icon" >
        <iconset resource="../Resources.qrc" >
         <normaloff>:/toolButton/images/home.png</normaloff>:/toolButton/images/home.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_MyWeiboPage" >
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>我的微博</string>
       </property>
       <property name="text" >
        <string>My Statuses</string>
       </property>
       <property name="icon" >
        <iconset resource="../Resources.qrc" >
         <normaloff>:/toolButton/images/myweibo.png</normaloff>:/toolButton/images/myweibo.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_AtMePage" >
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>@我的</string>
       </property>
       <property name="text" >
        <string>AtMe</string>
       </property>
       <property name="icon" >
        <iconset resource="../Resources.qrc" >
         <normaloff>:/toolButton/images/atme.png</normaloff>:/toolButton/images/atme.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_CommentPage" >
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>评论</string>
       </property>
       <property name="text" >
        <string>Comments</string>
       </property>
       <property name="icon" >
        <iconset resource="../Resources.qrc" >
         <normaloff>:/toolButton/images/comment.png</normaloff>:/toolButton/images/comment.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_FavouritePage" >
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>我的收藏</string>
       </property>
       <property name="text" >
        <string>MyFavourite</string>
       </property>
       <property name="icon" >
        <iconset resource="../Resources.qrc" >
         <normaloff>:/toolButton/images/favourite.png</normaloff>:/toolButton/images/favourite.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_MessagePage" >
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>私信</string>
       </property>
       <property name="text" >
        <string>Message</string>
       </property>
       <property name="icon" >
        <iconset resource="../Resources.qrc" >
         <normaloff>:/toolButton/images/message.png</normaloff>:/toolButton/images/message.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_configure" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>设置</string>
       </property>
       <property name="text" >
        <string>conf</string>
       </property>
       <property name="icon" >
        <iconset resource="../Resources.qrc" >
         <normaloff>:/toolButton/images/configure.png</normaloff>:/toolButton/images/configure.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_more" >
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>19</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>19</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>更多</string>
       </property>
       <property name="text" >
        <string>more</string>
       </property>
       <property name="icon" >
        <iconset resource="../Resources.qrc" >
         <normaloff>:/toolButton/images/more.png</normaloff>:/toolButton/images/more.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>19</height>
        </size>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources.qrc" />
 </resources>
 <connections/>
 <slots>
  <slot>homeButtonClicked()</slot>
 </slots>
</ui>
